README.md

```diff
@@ -27,3 +27,4 @@ See the [AutoGluon Website](http://autogluon.mxnet.io/index.html) for instructio
 ## License
 
 This library is licensed under the Apache 2.0 License.
+
```

Comment:
Why adding an empty line here

Reply:
We should end all files with a new line, thats what this adds.